community_supplied/Interface/check-ethernet-pause-frame-counters-netconf.rule

```diff
@@ -0,0 +1,188 @@
+/*
+* Detects interface ethernet pause frame counters and notifies when
+* anomalies are found.
+*
+* One input control detection
+*
+* 1) interface-name, is a regular expression that matches the
+* interfaces that you would like to monitor. By default it's
+* 'ge-.*|xe-.*|mge-.*|et-.*?' which matches all interfaces. Use
+* something like 'ge.*' to match only gigabit ethernet interfaces.
+*/
+healthbot {
+    topic interface.statistics {
+        description "Monitors and notify interface statistics i.e. link state, flaps, neighbor state, input & output traffic and errors";
+        synopsis "Interface statistics analyzer";
+        rule check-ethernet-pause-frame-counters-netconf {
+            keys name;
+            synopsis "Ethernet pause frame counter analyzer";
+            description "Collects the ethernet pause frame counter periodically and notifies in case of anomalies";
+            /*
+            * Monitors the input and output pause frame counters and
+            * notifies via the dashboard colors when the input or the
+            * output pause frame count increases by at least by 1 in
+            * 5m time range to red. Otherwise it is set to green.
+            *
+            * Use name as key for rule
+            *
+            *
+            * Sensor Configuration to get data from network devices.
+            *
+            */
+            sensor interface {
+                synopsis "Interfaces netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables & views to collect data from network device";
+                iAgent {
+                    file IntStatsTable.yml;
+                    table IntStatsTable;
+                    frequency 60s;
+                }
+            }
+            /*
+            * Fields defined using sensor path. Map the longer sensor names
+            * to the shorter field names used in the rules.
+            */
+            field admin-status {
```

Comment:
"admin-status" field not being used. You can filter admin down interfaces using where condition in field "interface-name"